resources/aws/environmentDefaults.go

```diff
@@ -101,14 +101,14 @@ func sandboxDefault() environmentDefault {
 			defaultInstanceType:            "t3.medium",
 			defaultInstanceProfileName:     "ec2InstanceRole",
 			defaultARMInstanceType:         "t4g.medium",
-			defaultInstanceStorageSize:     200,
+			defaultInstanceStorageSize:     50,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should this go in a separate PR?

Reply:
Yes it should, I have a local commit that I forgot to push removing this change, thanks!